spotify_player/src/config/theme.rs

```diff
@@ -74,6 +77,7 @@ pub struct ComponentStyle {
     pub playlist_desc: Option<Style>,
     pub table_header: Option<Style>,
     pub selection: Option<Style>,
+    pub secondary_row: Option<Style>,
```

Comment:
I named this secondary row because I thought it could be used in more places than just the help table, but let me know if it should just be for the help table.